### PR TITLE
Background Blur added across multiple Input Fields in Edit Popups

### LINKED ISSUE #5644: Add Background Blur to Input Fields in Edit Popups
# Current behavior

Some input fields lack a backdrop filter and have a plain background color.

Current:
CleanShot 2024-05-29 at 10 52 59

Desired:
CleanShot 2024-03-12 at 17 20 10

# Desired Behavior

Add the background blur to every field edit popup

```
backdrop-filter: blur(12px) saturate(200%) contrast(50%) brightness(130%)
background: rgba(252, 252, 252, 0.5)
```

Fields to update (but do check if I missed any):

- Calendar field
- Domain name
- Address
- Json
- Left part of the phone number field

CleanShot 2024-05-29 at 10 50 59

### PR DESCRIPTION
**Pull request by shyamsundertard**
Fix #5644 

**Apply background blur to input fields across UI for improved consistency and appearance with Theme compatibility :**

1.  Updated **TextAreaInput.tsx** to apply background blur to text input fields (e.g., City name).

2. Enhanced **PhonesFieldInput.tsx** to apply background blur for seamless styling in phone number input sections.

3. Modified **TextInputV2.tsx** to add background blur in address inputs, workspace fields (e.g., Blocklist, Name fields), invitation inputs, profile name/email fields, and other input areas.

4. Updated **DateTimeInput.tsx** and **DateInput.tsx** to improve the appearance of date, time, and calendar popup fields with background blur.

5. Refined **DropdownMenuInput.tsx** and **DropdownMenuSearchInput.tsx** to add background blur to inputs for Emails, Domain name, and search fields across the web (e.g., Time Zone dropdowns, phone number country search).

Some images are attached for reference.

1_b
1_w
2_b
2_w
3_b
3_w
4_b
4_w
5_b
5_w
6_b
6_w
7_b
7_w
8_b
8_w
9_b
9_w
10_b
10_w
11_b
11_w
12_b
12_w

### CODE REVIEW

**Comment by Bonapara** [comment]
Thanks for your PR @shyamsundertard! Here is some feedback:

# Blurred Area

Only the **floating inputs** should have a blurred background. Standard inputs, such as those in the settings, shouldn't have a blurred background.

# Phone Field

### Your Branch

Why is there a white background on the new phone input? It seems the phone input was already fixed on the main branch:

CleanShot 2024-11-01 at 15 05 38@2x

### Main

No white background (desired behavior)

CleanShot 2024-11-01 at 15 07 27@2x

# Link, LinkedIn, Phone, Domains...

### Your Branch

We can see the cell border underneath:

CleanShot 2024-11-01 at 15 09 47@2x

### Desired Behavior

The field should open directly above the cell:

CleanShot 2024-11-01 at 15 10 40@2x

**Comment by shyamsundertard** [comment]
@Bonapara 

**Changes have been made so that only the floating inputs have a blurred background**
but before committing them I would like to clear some queries:

**Regarding white background on the new phone input:**
Screenshot 2024-11-02 at 6 32 21 PM

There is no visible changes in **Light Theme** but in **Dark Theme** it was required as Phone number left part had a container but number input field did't had , so a common container style was added:
**Before:**
Screenshot 2024-11-01 at 8 51 26 PM

**After:**
2_b

Please let me know if this common background is required.

**Regarding Cell border visibility:**
Screenshot 2024-11-02 at 6 27 25 PM

This is not affected by my commit and I also tried to know cause of it and I got to know it can be because of the changes made in PR #8256 , image is attached for reference:
Screenshot 2024-11-02 at 6 44 09 PM

some other changes are also made for Dropdowns in PR #8256

**Comment by Bonapara** [comment]
Hi @shyamsundertard, 

Regarding the blur, yes we want to avoid introducing a "plain" background and keep the "before" behavior.

Regarding the position, could you make sure to fix it even if it's not from your commit? Thanks a lot!

**Comment by shyamsundertard** [comment]
Sure @Bonapara , I'll make sure to fix that also.

**Comment by shyamsundertard** [comment]
@Bonapara suggested changes have been committed, which includes blurred background applied only to floating inputs and fixed the cell border visibility issue: 
Screenshot 2024-11-10 at 8 19 32 PM
Screenshot 2024-11-10 at 8 20 47 PM

Please let me know if there's anything further I should modify.

**Comment by Bonapara** [comment]
Seems the cell border visibility issue has been resolved in another commit. Can you pull main on your branch?

**Comment by shyamsundertard** [comment]
@Bonapara is there anything else to modify ? As PR is not yet merged.

**Comment by Bonapara** [comment]
Looking good to me but I'll let @lucasbordeau do the review and check if everything is fine!

File: packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx [hunk 1]
```diff
@@ -22,11 +22,10 @@ const StyledDropdownMenu = styled.div<{
 
   display: flex;
 
-  height: fit-content;
-
   flex-direction: column;
   z-index: 30;
-  width: ${({ width = 200 }) =>
+  overflow-y: 30;
```
Comment (lucasbordeau) [inline packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx:27]: Please revert this we don't want to modify this component.

File: packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenuItemsContainer.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ const StyledDropdownMenuItemsExternalContainer = styled.div<{
   flex-direction: column;
   gap: 2px;
   max-height: ${({ hasMaxHeight }) => (hasMaxHeight ? '188px' : 'none')};
+  overflow-y: auto;
```
Comment (lucasbordeau) [inline packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenuItemsContainer.tsx:16]: Please don't modify this component

**Comment by shyamsundertard** [comment]
@lucasbordeau I have reverted above suggested changes.

**Review by lucasbordeau — APPROVED** [review]

**Pull request merged**